playbooks/byo/preflight/action_plugins/verify_openshift_image.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+
+from ansible.plugins.action import ActionBase
+
+
+class ActionModule(ActionBase):
+
+    def run(self, tmp=None, task_vars=None):
+        ''' handler verifying openshift images are available '''
+        if task_vars is None:
+            task_vars = dict()
```

Comment:
Since we don't otherwise use `task_vars`, we don't have to do the defaulting here, `_execute_module` already does that:
https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/action/__init__.py#L520-L525